src/BinaryParsers/ElfBinary/ElfBinary.cs

```diff
@@ -217,6 +236,42 @@ public ulong NormalizeAddress(ulong address)
                 : address - CodeSegmentOffset;
         }
 
+        /// <summary>
+        /// Get if section exists
+        /// </summary>
+        public bool SectionExists(string sectionName)
```

Comment:
>public bool SectionExists(string sectionName) [](http://example.com/codeflow?start=8&length=45)

Can we make this helper methods private?

(this applies for all methods below that u added)